runtime/calamari/src/lib.rs

```diff
@@ -571,6 +575,23 @@ impl pallet_scheduler::Config for Runtime {
 	type MaxScheduledPerBlock = MaxScheduledPerBlock;
 	type WeightInfo = weights::pallet_scheduler::SubstrateWeight<Runtime>;
 	type OriginPrivilegeCmp = OriginPrivilegeCmp;
+	type PreimageProvider = Preimage;
+	type NoPreimagePostponement = NoPreimagePostponement;
+}
+
+parameter_types! {
+	pub const PreimageMaxSize: u32 = 4096 * 1024;
```

Comment:
Does it mean 4MB?

Reply:
This must be bytes, everything else is per byte.

Reply:
@Dengjianping do you have an idea about this ? Not really clear why this was chosen by parity and everyone copied.